src/main/resources/lang_en.properties

```diff
@@ -3,7 +3,7 @@ lang.login=Login
 lang.password=Password
 lang.logIn=Log in
 lang.loginPageEntry=Welcome!
-lang.signIn=Log up
+lang.signIn=Sign in
 lang.logOut=Log out
 lang.currentUser=You - 
```

Comment:
what the `-`?

Reply:
`Log up` means to write some sort of travelled distance on plane or car. `Sign in` means to sign up for something or register.

Reply:
`lang.currentUser=You -`

Reply:
It is a `-` character, so final string would look like You - AdminName.
We probably could just remove this from lang.bundle as we don't use it anywhere beside admin.jsp, and we don't need admin.jsp anymore because we have navigation panel.

Reply:
you can save only You, but remove `-`. Properties only for words

Reply:
Changed it to `lang.currentUser=You`.